.github/workflows/build.yml

```diff
@@ -37,7 +37,7 @@ jobs:
     name: "Build and test on aarch64"
     strategy:
       matrix:
-        pyver: [cp37-cp37m, cp38-cp38, cp39-cp39]
+        pyver: [cp38-cp38, cp311-cp311]
```

Comment:
Is it OK to change from `cp37-cp37m` to `cp38-cp38`, as `cp38-cp38m`is not available?
```
 + /opt/python/cp38-cp38m/bin/python -m venv .env
bash: /opt/python/cp38-cp38m/bin/python: No such file or directory
Error: Process completed with exit code 1.
```

Reply:
Yes, it is